build/templates/session.py.mako

```diff
@@ -195,12 +195,12 @@ constructor_params = helper.filter_parameters(init_function, helper.ParameterUsa
         self._encoding = encoding
 
         # Store the parameter list for later printing in __repr__
```

Comment:
Is this comment about to be outdated because you are about to use it for other things?
If yes, consider making the comment a bit more generic:

```suggestion
# Store the parameter list for later uses, such as printing in __repr__.
```